sql/src/test/java/org/opensearch/sql/sql/parser/AstExpressionBuilderTest.java

```diff
@@ -520,6 +520,8 @@ public void relevanceSimple_query_string() {
             + "analyzer='keyword', operator='AND')"));
   }
 
+
+  //TODO: Make version of this for query()
```

Comment:
Think you can remove this now.

Reply:
Fixed in fe67f0168906e58e0b61e1f3f6f9072f14c19b67